go/cmd/dolt/commands/tblcmds/import.go

```diff
@@ -477,11 +477,20 @@ func newImportDataMover(ctx context.Context, root *doltdb.RootValue, dEnv *env.D
 	var wr table.TableWriteCloser
 	switch impOpts.operation {
 	case CreateOp:
-		wr, err = impOpts.dest.NewCreatingWriter(ctx, impOpts, dEnv, root, srcIsSorted, wrSch, statsCB, opts)
+		filePath, _ := dEnv.FS.Abs(impOpts.SrcName())
```

Comment:
You are discarding the error return value here -- the one you are checking below is from the outer scope

Same comment a couple lines down